test/elpy-config-test.el

```diff
@@ -21,4 +21,4 @@
                                          (elpy/wait-for-output "Only")
                                          (buffer-string))))
                            (should (equal python-check-command "/foo/bar/flake8"))
-                           (should (string-match "python -m pip install.*flake8" output))))))
+                           (should (string-match ".*python.* -m pip install.*flake8" output))))))
```

Comment:
Is the `.*` at the beginning really useful ?
`string-match` should return non-nil for a regexp anywhere in the string.

Reply:
You're right.  I've fixed this in 766ddf9 and additionally tightened the regex to account for the space between the hungry glob and -m.

Reply:
weird, my comments aren't showing up here... 'hope you got them elsewhere!